### ISSUE TITLE
Placing the king requires teleportation in the weapon set.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmith/hedgewars#82]
```
What steps will reproduce the problem?
1. Play with king.
2. Use weapon set with no teleportation.
3. Run the fight.

What is the expected output? What do you see instead?
I expected that the king will use the extra teleport, as it happens, when you 
choose to place all hedgehogs manually, but the king uses the teleport from 
weapon set (if there is any). I think there should be an ability to choose if 
you want to place (only) the king manually before starting the game.

What version of the product are you using? On what operating system?
0.9.13 on Windows XP SP2

Please provide any additional information below.


```

-----
Original issue reported on code.google.com by `adibiaz...@gmail.com` on 7 Oct 2010 at 4:10

**Comment by GoogleCodeExporter** [comment]
```
Meh. If you want that, use a weaponset with 1 teleport.

If you don't, don't.

I'm going to mark this WONTFIX unless you can marshal a decent argument for the 
extra clutter.

```

-----
Original comment by `kyberneticist@gmail.com` on 7 Oct 2010 at 7:19

**Comment by GoogleCodeExporter** [comment]
```
I don't particularly consider this a bug, and has been sitting here for a 
while.  Marking WontFix.

```

-----
Original comment by `kyberneticist@gmail.com` on 29 Oct 2010 at 3:01
* Changed state: **WontFix**
